controls/scheduler/built-in-dialogs.md

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
The page doesn't have an inline ToC ("In this article", in the sidebar). That's due to the heading levels. Consider switching to H2s for Methods, Events, and Customization.